### ISSUE TITLE
moto one 5G cellular issues (updated)

### ISSUE THREAD

**Issue by truvision-dude** [phhusson/treble_experimentations#2079]
After some research and a lot of testing (5 hours to be precise), I think I have pinpointed my cellular issue

So basically, here is what happens:
- When on stock android 11, everything works properly
- When both android 12 and android 11 GSIs are flashed, cellular does not work
- When the phone is first booted up in the GSI, cellular connects for 5 seconds (3-5 bars) which is what is supposed to happen in my area, but after 5 seconds, it drops the connection (the cellular icon shows an ! next to it)
- For the rest of setup, cellular issues prevail, and I am forced to use Wi-Fi. Choosing to use mobile data just causes it to loop.
- After setup, when going into settings and changing my cellular connection to LTE/UMTS, I get connected to xfinity mobile. I am able to use data, however, calling and Wi-Fi do not work, no 5G, and much lower signal then what I usually get

Now here is where things get interesting, when I install the ims apk in phh-treble settings and reboot, it says no sim and in cellular settings, it says tap to activate sim card (which does nothing), however, if I set up the phone as usual, don't change the preferred network type, and install the ims apk, I get the VoLTE settings and Wi-Fi calling. However, there is still no signal and calling/text does not work. The interesting thing is when I connect to Wi-Fi, it shows half bars of service, but still says no service in sim info. When I change the preferred network setting after ims apk is installed and reboot, it says no sim card again.

So to recap:
- For the first 3-5 seconds of setup, there appears to be a working connection, but it drops
- The ims works as its supposed to (I think) until I change the preferred network setting
- Changing the preferred network setting does yield some success, but no 5G and no calling/texting
- Xfinity mobile runs on verizon, and the sim originally detects itself as a verizon sim, which could be the cause of this problem

So does anyone have a solution for my issue? This is not just on the android 12 gsi, its on the android 11 one too. On motorola's stock android, everything works fine.

**Comment by truvision-dude** [comment]
@phhusson will you need the logcat?

**Comment by ponces** [comment]
Yes, send logcat logs please

**Comment by truvision-dude** [comment]
This is the logcat from the first boot of the ponces gsi (both android 12 and 11 yield the same results) as soon as I plugged the sim card in. As I have said before, on first boot and restarts, it connects to xfinity for 4-5 seconds, and drops to no service
[logcat.txt](https://github.com/phhusson/treble_experimentations/files/7630605/logcat.txt)

Also, NFC does not work, if you can find a way to fix it, I will be very grateful.

Thanks

**Comment by truvision-dude** [comment]
Any updates?
